.gitignore

```diff
@@ -30,4 +30,6 @@ erd.pdf
 
 secrets.zip
 
+record.csv
```

Comment:
This is redundant; this has already been done in `master`

Reply:
fixed